### ISSUE TITLE
Pretrained model for CIFAR 10 dataset?

### ISSUE THREAD

**Issue by tianweiy** [facebookresearch/pytorch_GAN_zoo#63]
Is there any pretrained model for cifar 10 dataset? I tried to train using the code here but the loss diverges in the end and I can only get 5.2 inception score.

**Comment by Molugan** [comment]
Hi,
Sorry for the delay.

There isn't a pretained model for CIFAR yet. But I can have a look.

**Comment by Molugan** [comment]
I've found a bug explaining partly the low score on CIFAR so far, after https://github.com/facebookresearch/pytorch_GAN_zoo/pull/72 you should easily get a 6.1 score. I'm working on the parameters to go up to 8.

**Comment by tianweiy** [comment]
Thanks. No hurry, I am using gan in TensorFlow in the meantime.

**Comment by heartInsert** [comment]
> Thanks. No hurry, I am using gan in TensorFlow in the meantime.

How  much  time  do  you  cost   in  training in cifar10  with  5.2  inception score ?

**Comment by tianweiy** [comment]
The 5.2 is trying to say that the loss diverges in the end. I suspect you can get higher inception score by training less time. It took about 2~3 days using 2 GPUs if I remember correctly.

**Comment by heartInsert** [comment]
God,  it 's  too  much  time for  me

**Comment by yueruchen** [comment]
@Molugan Can the current version reproduce the results for cifar-10?

**Comment by JanineCHEN** [comment]
> @Molugan Can the current version reproduce the results (fid=8.80) for cifar-10?

@Molugan Having same question here, any insight would be greatly appreciated!
